lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgid "Unloading filament"
+msgstr "Tar bort filament"
+
+# MSG_TOTAL c=6
+#: messages.c:100
+msgid "Total"
+msgstr "Total"
+
+# MSG_USED c=19
+#: ultralcd.cpp:5928
+msgid "Used during print"
+msgstr "Anv\xe1nds under utskr"
+
+# MSG_MENU_VOLTAGES c=15
+#: ultralcd.cpp:2050
+msgid "Voltages"
+msgstr "Sp\xe1nning"
+
+# MSG_UNKNOWN c=13
+#: ultralcd.cpp:2005
+msgid "unknown"
+msgstr "ok\xe1nd"
+
+# MSG_USERWAIT c=20
+#: Marlin_main.cpp:4347
+msgid "Wait for user..."
+msgstr "Inv\xe1ntar anv\xe1ndare."
+
+# MSG_WAITING_TEMP c=20 r=4
+#: ultralcd.cpp:3283
+msgid "Waiting for nozzle and bed cooling"
+msgstr "V\xe1ntar pa munstycks- och b\xe1ddkylning"
+
+# MSG_WAITING_TEMP_PINDA c=20 r=3
+#: ultralcd.cpp:3244
+msgid "Waiting for PINDA probe cooling"
+msgstr "V\xe1ntar pa PINDA-sondens kylning"
+
+# MSG_CHANGED_BOTH c=20 r=4
+#: Marlin_main.cpp:1597
+msgid "Warning: both printer type and motherboard type changed."
+msgstr "Varning: bade skrivartyp och moderkortstyp har \xe1ndrats."
+
+# MSG_CHANGED_MOTHERBOARD c=20 r=4
+#: Marlin_main.cpp:1589
+msgid "Warning: motherboard type changed."
+msgstr "Varning: moderkortstyp \xe1ndrad."
+
+# MSG_CHANGED_PRINTER c=20 r=4
+#: Marlin_main.cpp:1593
+msgid "Warning: printer type changed."
+msgstr "Varning: skrivartyp har \xe1ndrats."
+
+# MSG_UNLOAD_SUCCESSFUL c=20 r=2
+#: Marlin_main.cpp:3789
+msgid "Was filament unload successful?"
+msgstr "Lyckades filamentutmatningen?"
+
+# MSG_SELFTEST_WIRINGERROR c=18
+#: messages.c:98
+msgid "Wiring error"
+msgstr "Kabelfel"
+
+# MSG_WIZARD c=17
+#: ultralcd.cpp:5771
+msgid "Wizard"
+msgstr "Guide"
+
+# MSG_XYZ_DETAILS c=18
+#: ultralcd.cpp:2039
+msgid "XYZ cal. details"
+msgstr "XYZ kal. detaljer"
+
+# MSG_BED_SKEW_OFFSET_DETECTION_FITTING_FAILED c=20 r=8
+#: messages.c:18
+msgid "XYZ calibration failed. Please consult the manual."
+msgstr "XYZ-kalibrering misslyckades. Se bruksanvisningen."
+
+# MSG_YES c=3
+#: messages.c:123
+msgid "Yes"
+msgstr "Ja"
+
+# MSG_WIZARD_QUIT c=20 r=8
+#: messages.c:120
+msgid "You can always resume the Wizard from Calibration -> Wizard."
+msgstr "Du kan alltid ateruppta guiden fran Kalibrering -> Guide."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_SKEW_EXTREME c=20 r=8
+#: ultralcd.cpp:3743
+msgid "XYZ calibration all right. Skew will be corrected automatically."
+msgstr "XYZ-kalibrering \xe1r ok. Skevhet kommer att korrigeras automatiskt."
+
+# MSG_BED_SKEW_OFFSET_DETECTION_SKEW_MILD c=20 r=8
+#: ultralcd.cpp:3740
+msgid "XYZ calibration all right. X/Y axes are slightly skewed. Good job!"
+msgstr "XYZ-kalibrering ok. X/Y-axlarna \xe1r mycket lite skeva. Bra jobbat!"
+
+# MSG_TIMEOUT c=12
+#: messages.c:157
+msgid "Timeout"
+msgstr "Timeout"
+
+# MSG_X_CORRECTION c=13
+#: ultralcd.cpp:5086
+msgid "X-correct:"
+msgstr "X-korrektion:"
+
+# MSG_BED_SKEW_OFFSET_DETECTION_PERFECT c=20 r=8
+#: ultralcd.cpp:3737
+msgid "XYZ calibration ok. X/Y axes are perpendicular. Congratulations!"
+msgstr "XYZ-kalibrering ok. X/Y-axlarna ar vinkelr\xe1ta. Grattis!"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.